src/main/java/com/jprotractor/NgWebDriver.java

```diff
@@ -11,137 +10,125 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.internal.WrapsDriver;
 
-public class NgWebDriver implements WebDriver, WrapsDriver {
-    private WebDriver driver;
-    private JavascriptExecutor jsExecutor;
-    private String rootElement;
-    @SuppressWarnings("unused")
-    private NgModule[] mockModules;
-    public boolean ignoresync;
```

Comment:
it will be difficult to test it if it is not available. the test is actually elementary - any non-Angular page will bomb to findElement and all other methods calling waitForAngular, i will provide tests tomorrow, before merging your code